todo_app/app.py

```diff
@@ -1,15 +1,31 @@
-from flask import Flask
+from flask import Flask, render_template, request
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import *
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return render_template('index.html', items=get_items())
 
+@app.route('/addItem', methods = ['POST'])
+def addItem():
+    add_item(request.form.get('name'))
+    return render_template('index.html', items=get_items())
```

Comment:
It's nicer to `return redirect('/')`
- Re-use code in the index function
- The browser will show the correct URL
- The user can refresh the page without resubmitting the form

It looks like you're using javascript to effectively achieve the same thing for the delete endpoint. If you change that to a redirect, you can remove the javascript from your index.html

Reply:
This has been done